tests/contrib/anthropic/test_anthropic_llmobs.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from tests.llmobs._utils import _expected_llmobs_llm_span_event
+
+
+@pytest.mark.parametrize(
+    "ddtrace_global_config", [dict(_llmobs_enabled=True, _llmobs_sample_rate=1.0, _llmobs_ml_app="<ml-app-name>")]
+)
+class TestLLMObsAnthropic:
```

Comment:
Can we add a test case for completion error (wrong API key is a good error case)?

Reply:
done